var/spack/repos/builtin/packages/migraphx/package.py

```diff
@@ -131,10 +131,9 @@ def cmake_args(self):
             )
         return args
 
-    def test(self):
+    def test_drivers(self):
+        """Test drivers"""
         if self.spec.satisfies("@:5.5.0"):
-            print("Skipping: stand-alone tests")
-            return
-        test_dir = join_path(self.spec["migraphx"].prefix, "bin")
-        with working_dir(test_dir, create=True):
-            self.run_test("UnitTests")
+            raise SkipTest("Package must be installed as version @5.5.1 or later")
+        exe = which("migraphx-driver", "migraphx-hiprtc-driver")
+        exe()
```

Comment:
This doesn't look like a conversion of the original implementation but a re-write.  Does the original not work (after a re-write)?